content/02.main-text.md

```diff
@@ -1,8 +1,11 @@
 ## Introduction {.page_break_before}
 
-Openness in research benefits the researchers who practice it [@doi:10.7554/eLife.16800], their scientific peers, and the public.
-Improving the availability of code and data helps ensure the methods are transparent and the results can be reproduced  [@doi:10.1038/s41562-016-0021].
-Publishing in an open access journal increases the number of readers and reviewers [@doi:10.1371/journal.pone.0000308].
+The internet enables science to be shared in real-time at a low cost to a global audience.
```

Comment:
I like this intro paragraph much better.  It immediately focuses on the opportunity / problem that Manubot targets.